src/views/Main.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div class="main">
+    <Header />
+  </div>
+</template>
+
+<script>
+import Header from '../components/Header.component.vue'
```

Comment:
제 로컬에서는 여기서 에러가 나는데, 파일명은 header.component.vue 인데 Header로 되어있어서 안 되는 것 같아요
```suggestion
import Header from '../components/header.component.vue'
```

Reply:
에러랑은 별개로 Vue 컴포넌트 파일은 파스칼 케이스로 작성하는게 일반적이고, .component 를 붙이는 것은 처음보는 건데 저희는 안 쓰고 있으니까 때면 좋을 것 같아요!
```suggestion
import Header from '../components/Header.vue'
```